cockatrice/src/chatview.h

```diff
@@ -44,6 +44,7 @@ private slots:
     ChatView(const TabSupervisor *_tabSupervisor, TabGame *_game, bool _showTimestamps, QWidget *parent = 0);
     void retranslateUi();
     void appendHtml(const QString &html);
```

Comment:
Maybe we should make `appendHtml` private and only expose server message and chat? Is there anywhere else in public context that uses that method?